flac/decode.go

```diff
@@ -138,8 +145,11 @@ func (d *decoder) Position() int {
 	return d.pos
 }
 
+// p represents flac sample num perhaps?
```

Comment:
I left this comment because it looked like the API for seek has not really been determined?  I'm not sure if sample number is really what you had in mind.